src/bin/electrs.rs

```diff
@@ -142,10 +145,22 @@ fn run_server(config: Arc<Config>) -> Result<()> {
     Ok(())
 }
 
-fn main() {
+fn main_() {
     let config = Arc::new(Config::from_args());
     if let Err(e) = run_server(config) {
         error!("server failed: {}", e.display_chain());
         process::exit(1);
     }
 }
+
+#[cfg(not(feature = "otlp-tracing"))]
```

Comment:
I know I originally suggested this code :sweat_smile: But I think that since we must have a negative `no-otlp-tracing` feature, it would be better to use it here too.

Reply:
done